perfrunbook/debug_hw_perf.md

```diff
@@ -213,7 +213,8 @@ On Ubuntu to enable SPE requires four extra steps
 # Install the arm_spe_pmu.ko module
 sudo apt install linux-modules-extra-$(uname -r)
 
-# Add kpti=off to the kernel boot command line
+# Add kpti=off to the kernel boot command line: GRUB_CMDLINE_LINUX, to set it for all the installed kernels.
```

Comment:
"Modify the kernel boot arguments for all installed kernels by adding `kpti=off` to GRUB_CMDLINE_LINUX in /etc/default/grub"

Reply:
Can we maybe provide a quick inline change cmd to it? something like:
```
sed -i -E 's/^(GRUB_CMDLINE_LINUX=".*)"/\1 kpti=off"/g'  /etc/default/grub
```

Reply:
The `sed` command is tricky to get right in all situations, e.g. what if the line already has `kpti=on`? It's better to explain what to do so the user knows where/what to modify, rather than copy/paste something hard to understand and potentially faulty.